packages/telegram-stamper/src/index.ts

```diff
@@ -0,0 +1,185 @@
+/// <reference lib="dom" />
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { TelegramStamperError } from "./errors";
+
+declare global {
+  interface Window {
+    Telegram: any;
+  }
+}
+
+// config when telegram stamper is passed in an api key
+export type TTelegramStamperConfig = {
+  apiPublicKey: string;
+  apiPrivateKey: string;
+  cloudStorageKeySuffix?: string;
```

Comment:
let's remove this option and replace it with "cloudStorageKey" to give customers full control over where keys are stored?

Reply:
Changed to cloudStorageKey